docs/dev/stream/operators/process_function.md

```diff
@@ -277,19 +277,18 @@ Both types of timers (processing-time and event-time) are internally maintained
 
 The `TimerService` deduplicates timers per key and timestamp, i.e., there is at most one timer per key and timestamp. If multiple timers are registered for the same timestamp, the `onTimer()` method will be called just once.
```

Comment:
Can we move this entire `Fault Tolerance` section to `dev/stream/state/checkpointing.md` for a new `Timers` section? This is a general concept that is not only relevant for `ProcessFunction`.

Reply:
Writing more docs about this might be a bigger change but moving 10 lines into a subsection on a different page can be done in this PR.